src/tools/compiletest/src/runtest.rs

```diff
@@ -2369,6 +2370,33 @@ impl<'test> TestCx<'test> {
         }
     }
 
+    /// Adds rmc scripts directory to the `PATH` environment variable.
+    fn add_rmc_dir_to_path(&self, rmc: &mut Command) {
+        // If the PATH enviornment variable is already defined,
+        if let Some((key, val)) = env::vars().find(|(key, _)| key == "PATH") {
+            // Add the RMC scripts directory to the PATH.
+            rmc.env(key, format!("{}:{}", self.config.rmc_dir_path.to_str().unwrap(), val));
+        } else {
+            // Otherwise, insert PATH as a new enviornment variable and set its value to the RMC scripts directory.
+            rmc.env(String::from("PATH"), String::from(self.config.rmc_dir_path.to_str().unwrap()));
+        }
+    }
+
+    /// Runs rmc model checker on the test file specified by `self.testpaths.file`. A "verification failed!" error is printed to the stdout if verfication fails.
+    fn run_rmc_test(&self) {
+        // Other modes call self.compile_test(...). However, we cannot call it here for two reasons:
+        // 1. It calls rustc instead of rmc, and
+        // 2. It may pass some options that do not make sense for rmc.
+        // So, we create our own command to execute rmc and pass it to self.compose_and_run_compiler(...) directly.
```

Comment:
```suggestion
        // Other modes call self.compile_test(...). However, we cannot call it here for two reasons:
        // 1. It calls rustc instead of RMC
        // 2. It may pass some options that do not make sense for RMC
        // So we create our own command to execute RMC and pass it to self.compose_and_run_compiler(...) directly.
```